### PR TITLE
first pass at fixing broken links

### PR DESCRIPTION
**Pull request by oxelson**

### CODE REVIEW

**Comment by oxelson** [comment]
@lesserwhirls First pass at fixing bad links in TDS 5.x docs.

File: docs/src/public/userguide/index.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ The TDS is developed and supported by Unidata, a division of the University Corp
 
 Some of the technology in the TDS:
 
-* THREDDS [Dataset Inventory Catalogs](updateme) are used to provide virtual directories of available data and their associated metadata.
```
Comment (lesserwhirls) [inline docs/src/public/userguide/index.md:20]: lesserwhirls

File: docs/src/public/userguide/pages/tds_tutorial/tds_configuration/AddingOgcIsoServices.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Adding OGC/ISO Services
-last_updated: 2018-04-02
+last_updated: 2020--04-01
```
Comment (lesserwhirls) [inline docs/src/public/userguide/pages/tds_tutorial/tds_configuration/AddingOgcIsoServices.md:3]: looks like an extra `-` after the year.

**Comment by lesserwhirls** [comment]
Heck...all those `updateme` links...ugh.

Other than that those few things, we're using netcdf-java 5.4 snapshot on master for TDS, so we should make sure any links to the netcdf-java docs are updated from `/netcdf-java/5.[0,1,2,3]` &rarr; `netcdf-java/5.4`.

Thank you @oxelson !

**Comment by oxelson** [comment]
@lesserwhirls - will do. Thanks for checking!

**Comment by oxelson** [comment]
@lesserwhirls Change request complete.

**Comment by lesserwhirls** [comment]
Thanks @oxelson!

**Pull request merged**